example/bench/recipe.cpp

```diff
@@ -0,0 +1,54 @@
+#include <random>
+
+#include <benchmark_cell.hpp>
+#include <common_types.hpp>
+#include <time_sequence.hpp>
+
+#include "recipe.hpp"
+
+cell_size_type bench_recipe::num_cells() const {
+    return params_.num_cells;
+}
+
+arb::util::unique_any bench_recipe::get_cell_description(cell_gid_type gid) const {
+    using RNG = std::mt19937_64;
+    arb::benchmark_cell cell;
+    cell.realtime_ratio = params_.cell.realtime_ratio;
+    cell.time_sequence = arb::poisson_time_seq<RNG>(RNG(gid), 0, 1e-3*params_.cell.spike_freq_hz);
+    return std::move(cell);
```

Comment:
Is this needed? Isn't local variable and constructor elision part of the standard? (Which is why C++ is such a nightmare --- such a basic thing requires "checking the standard")

Reply:
It is needed, because we are not returning a value of type `arb::benchmark_cell`. Instead we are returning an `arb::unique_any`. When saying `return X;`, we are saying to create a `unique_any` constructed with `X`, which by default will use the copy constructor. By manually turning it into an rvalue reference, we get the move constructor.

Reply:
Probably should comment when you have to coerce the compiler like that.